packages/forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -204,6 +204,19 @@ exports.getAppealsCaseData = async (lpaCode) => {
 	return handler(`/api/v1/appeals-case-data/${lpaCode}`, 'GET');
 };
 
+exports.getAppealsCaseDataV2 = async (lpaCode) => {
+	const urlParams = new URLSearchParams();
+	urlParams.append('lpa-code', lpaCode);
+	return handler(`/api/v2/appeal-cases?${urlParams.toString()}`, 'GET');
+};
+
+exports.getDecidedAppealsCaseDataV2 = async (lpaCode) => {
+	const urlParams = new URLSearchParams();
+	urlParams.append('lpa-code', lpaCode);
+	urlParams.append('decided-only', 'true');
+	return handler(`/api/v2/appeal-cases?${urlParams.toString()}`, 'GET');
+};
+
```

Comment:
Perhaps one for a future PR, let's move these into the common `appeals-api-client` :)

Reply:
I'd like to cover this in a future PR - I think there are several functions which are ripe for moving to the common package

Reply:
I'll look at moving the date-sort function into common too.